packages/cli/src/util/createServer.ts

```diff
@@ -26,41 +27,48 @@ export async function createMultiProcessPrism(options: CreatePrismOptions) {
   } else {
     const logInstance = createLogger('CLI');
     try {
-      return await createPrismServerWithLogger(options, logInstance);
+      return createPrismServerWithLogger(options, logInstance).catch(() => cluster.worker.kill());
```

Comment:
I'd stay with the previous solution with `await`. Having two ways of catching errors is hard to read.

Reply:
Yeah this is one of these refactors that didn't really work :trollface: — will put it back.

Reply:
This's been done.